frontend/resourceadm/components/ResourceReferenceFields/ResourceReferenceFields.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import type { ResourceReference } from 'app-shared/types/ResourceAdm';
+import { ResourceReferenceFieldset } from './ResourceReferenceFieldset';
+import { FieldsetWrapper } from '../FieldsetWrapper';
+
+// Empty value for when adding a new field
+const emptyResourceReference: ResourceReference = {
+  referenceSource: 'Default',
+  reference: '',
+  referenceType: 'Default',
+};
+
+export type ResourceReferenceFieldsProps = {
+  /**
```

Comment:
We don't need to have documentation for each prop value in the definition of the prop type. It is enough with the documentation above the component (e.g., line 35-45 in this case). 
I know I've added it on the old files, but we can remove that going forward, and keep it on the component only 😄